### PR TITLE
Propose RFC: decorators

### PR DESCRIPTION
**Pull request by dhh1128**
Signed-off-by: Daniel Hardman <daniel.hardman@gmail.com>

### CODE REVIEW

File: concepts/0011-decorators/README.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+of __overlays__. These are layers of additional meaning that accumulate above a __schema
+base__. Decorators as described here are quite similar in intent. There are some subtle
+differences, though. The most interesting is that decorators as described here may be
+applied to things that are not schema-like (e.g., to a message family as a whole, or to
+a connection, not just to an individual message).
+
+We may be able to resolve these two worldviews, such that decorators are viewed as overlays
+and inherit some overlay goodness as a result. However, it is unlikely that decorators
+will change significantly in form or substance as a result. We thus believe the current
+mental model is already HIPE-worthy, and represents a reasonable foundation for immediate
```
Comment (kdenhartog) [inline concepts/0011-decorators/README.md]: Can you change this to RFC?

Comment (dhh1128) [reply concepts/0011-decorators/README.md]: Good catch. Fixed. I also fixed a couple other places that I hadn't caught.

File: concepts/0011-decorators/README.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+  that you should assume for all interactions." (This may or may not be useful. It introduces ambiguity--should settings declared
+  at this scope override those at the scope of a message family or message type? It
+  also adds a burden of remembering to another party; some of what informs the interaction
+  isn't present in the bits that flow later on. Thus, this style of decorator is also
+  left as a possibility for future exploration and is not actively supported at present.)
+
+## Reference
+
+This section of this RFC will be kept up-to-date with a list of globally accepted
+decorators, and links to the HIPEs that define them.
```
Comment (kdenhartog) [inline concepts/0011-decorators/README.md]: Should this be HIPE or RFC?

Comment (dhh1128) [reply concepts/0011-decorators/README.md]: fixed

File: concepts/0011-decorators/README.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+}
+```
+This chunk of JSON is defined independent of any particular message schema, but
+[is understood to be available in all DIDComm schemas](
+https://github.com/hyperledger/indy-hipe/blob/3d8ec6c522cacaaef20b3a999f3c75b5b1217b70/text/json-ld-compatibility/README.md#decorators
+).
+
+### Basic Conventions
+
+Decorators are defined in HIPEs that document a general pattern such as [message threading RFC](
```
Comment (kdenhartog) [inline concepts/0011-decorators/README.md]: Should this be HIPE or RFC?

**Pull request merged**